clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/DummyVdsNode.java

```diff
@@ -94,8 +95,8 @@ public void run() {
                     long currentTime = timer.getCurrentTimeInMillis();
                     for (Iterator<Req> it = waitingRequests.iterator(); it.hasNext(); ) {
                         Req r = it.next();
-                        if (r.timeout <= currentTime) {
-                            log.log(Level.FINE, () -> "Dummy node " + DummyVdsNode.this + ": Responding to node state request at time " + currentTime);
+                        if (currentTime >= r.timeToReply) {
+                            log.log(Level.INFO, () -> "Dummy node " + DummyVdsNode.this + ": Responding to node state request at time " + currentTime);
```

Comment:
I'm assuming this log level bump is to make debugging easier for spurious failures?

Reply:
No, not intended, will fix. Thanks